test/cql-pytest/conftest.py

```diff
@@ -221,3 +235,22 @@ def temp_workdir():
     """ Creates a temporary work directory, for the scope of a single test. """
     with tempfile.TemporaryDirectory() as workdir:
         yield workdir
+
+@pytest.fixture(scope="function")
+def xfail_tablets(test_keyspace):
+    if test_keyspace.endswith('_tablets'):
+        pytest.xfail("test fails with tablets")
```

Comment:
this is the wrong way to do xfail - it forces an "XFAIL" status and doesn't even try to run the test (i.e., no chance for XPASS). I'll send my own PR in an hour, and you can see the right way - basically
```
request.node.add_marker(pytest.mark.xfail(reason='Test expected to fail with tablets experimental feature on'))
```

Reply:
No need to send yet another PR, I already have mine ready: https://github.com/scylladb/scylladb/pull/16840